core-bundle/src/Resources/contao/library/Contao/Template.php

```diff
@@ -150,6 +150,22 @@ public function __set($strKey, $varValue)
 	 */
 	public function __get($strKey)
 	{
+		if ('stringUtil' === $strKey)
+		{
+			return new class()
```

Comment:
I guess we could ensure we do not create a new instance for every call but let's see for general feedback first.